packages/component/src/Activity/private/ThumbButton.tsx

```diff
@@ -57,7 +57,7 @@ const ThumbButton = memo(({ className, direction, disabled, onClick, pressed, ti
         direction={direction}
         filled={true}
       />
-      <Tooltip>{buttonTitle}</Tooltip>
+      <Tooltip className={className}>{buttonTitle}</Tooltip>
```

Comment:
We shouldn't reuse the same className for both: the tooltip and the button.

With the addition of the `--webchat__tooltip-anchor-inline-start` you should be able to have a class on the feedback form: `webchat__thumb-button--submitted`.

Which should roughly do:

```css
.webchat__thumb-button--submitted .webchat__tooltip {
  --webchat__tooltip-anchor-inline-start: 20%;
}
```